src/main/java/org/jitsi/videobridge/BitrateController.java

```diff
@@ -0,0 +1,568 @@
+ * actually delegated to the individual {@link SimulcastController} of each
+ * media stream track.
+ *
+ * @author George Politis
+ */
+public class BitrateController
+    implements TransformEngine
+{
+    /**
+     * An empty list instance.
+     */
+    private static final List<String> INITIAL_EMPTY_LIST
+        = Collections.unmodifiableList(new LinkedList<String>());
+
+    /**
+     * The {@link VideoChannel} which owns this {@link BitrateController}.
+     */
+    private final VideoChannel dest;
+
+    /**
+     * The bitrate controller of the {@link Conference}.
+     */
+    private final Map<Integer, SimulcastController>
+        subCtrls = new ConcurrentHashMap<>();
+
+    /**
+     * The {@link PacketTransformer} that handles incoming/outgoing RTP
+     * packets for this {@link BitrateController} instance. Internally,
+     * it delegates this responsibility to the appropriate media stream track
+     * bitrate controller ({@link SimulcastController}, etc).
+     */
+    private final PacketTransformer rtpTransformer = new RTPTransformer();
+
+    /**
+     * The {@link PacketTransformer} that handles incoming/outgoing RTCP
+     * packets for this {@link BitrateController} instance. Internally,
+     * it delegates this responsibility to the appropriate media stream track
+     * bitrate controller ({@link SimulcastController}, etc).
+     */
+    private final PacketTransformer rtcpTransformer = new RTCPTransformer();
+
+    /**
+     * The {@link List} of endpoints that are currently being forwarded,
+     * represented by their IDs. Required for backwards compatibility with
+     * existing LastN code.
+     */
+    private List<String> forwardedEndpoints = INITIAL_EMPTY_LIST;
+
+    /**
+     * Initializes a new {@link BitrateController} instance which is to
+     * belong to a particular {@link VideoChannel}.
+     *
+     * @param dest the {@link VideoChannel} that owns this instance.
+     */
+    public BitrateController(VideoChannel dest)
+    {
+        this.dest = dest;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public PacketTransformer getRTPTransformer()
+    {
+        return rtpTransformer;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public PacketTransformer getRTCPTransformer()
+    {
+        return rtcpTransformer;
+    }
+
+    /**
+     *
+     * @param data
+     * @param buf
+     * @param off
+     * @param len
+     * @param source
+     * @return
+     */
+    public boolean rtpTranslatorWillWrite(
+        boolean data, byte[] buf, int off, int len, RtpChannel source)
+    {
+        Integer ssrc;
+        if (data)
+        {
+
+            long ret = RawPacket.getSSRCAsLong(buf, off, len);
+            if (ret < 0)
+            {
+                return false;
+            }
+            ssrc = (int) ret;
+        }
+        else
+        {
+            // NOTE(gp) This will need to be adjusted when we enable
```

Comment:
should this comment be on the RTCPHeaderUtils.getSenderSSRC method itself? rather than where it's being called?